parsec/core/fs/storage/manifest_storage.py

```diff
@@ -88,6 +92,47 @@ async def _create_db(self):
                 );
                 """
             )
+            # Singleton storing the pattern_filter
+            cursor.execute(
```

Comment:
We don't have a migration system for the sqlite database. I guess we are lucky this time given we can work around with this `CREATE TABLE IF NOT EXISTS` ;-)